core/templates/pages/exploration-player-page/learner-experience/conversation-skin.directive.html

```diff
@@ -60,6 +60,19 @@
                       on-click-continue-button="showUpcomingCard()">
         </progress-nav>
       </div>
+      <div ng-if="collectionSummary && isOnTerminalCard() && isCurrentCardAtEndOfTranscript() && recommendedExplorationSummaries.length > 0" class="conversation-skin-next-lesson-wrapper">
```

Comment:
This is working fine for collections and normal explorations with recommendations, but for explorations in stories, this still doesn't work (since you're not handling that anywhere). For explorations in stories, it should work identical to those in collections.

Reply:
Note: You'd need to enable the `ENABLE_NEW_STRUCTURE_VIEWER_UPDATES` flag to test it for stories.

Reply:
Done